pkg/kncloudevents/v2/cloudevents.go

```diff
@@ -47,14 +48,14 @@ func NewCloudEventsClient(target string, ceOverrides *duckv1.CloudEventOverrides
 	return &client{
 		ceClient:    ceClient,
 		ceOverrides: ceOverrides,
-		reporter:    reporter,
+		reporter:    metrics.NewStatsReporterAdapter(reporter),
 	}, nil
 }
 
 type client struct {
```

Comment:
This client does specific things for Adapters. Please move it back to `adapter/v2`

Reply:
fixed.